docs/python.md

```diff
@@ -145,6 +150,27 @@ class Predictor(BasePredictor):
 
 Each of the output object's properties must be one of the supported output types. For the full list, see [Input and output types](#input-and-output-types).
 
+### Returning a list
+
+The `predict()` method can return a list of any of the supported output types. Here's an example that outputs multiple files:
+
+```py
+from cog import BasePredictor, Path
+
+class Predictor(BasePredictor):
+    def predict(self) -> List[Path]:
+        predictions = ["foo", "bar", "baz"]
+        output = []
+        for i, prediction in enumerate(predictions):
+            out_path = Path(f"/tmp/out-{i}.txt")
+            with out_path.open("w") as f:
+                f.write(prediction)
+            output.append(out_path)
+        return output
+```
+
+Files are named in the format `output.<index>.<extension>`, e.g. `output.0.txt`, `output.1.txt`, and `output.2.txt` from the example above.
```

Comment:
Not a blcoker to shipping, but this is a bit out of context -- nothing mentions `cog predict` anywhere.

Reply:
Maybe this should live in the CLI documentation, but is currently just `cog predict --help`.

Reply:
Ah yeah good point. Should we take this last line out and add a TODO on https://github.com/replicate/cog/issues/433?